moped-database/migrations/1666899604790_alter_table_public_moped_proj_personnel_drop_column_role_id/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."moped_proj_personnel" drop column "role_id" cascade;
```

Comment:
This migration is a bit tricky to test, since in local dev the migration runs before our seeds are applied, but in reality it will be applied to a hydrated database.

I have tested this by manually running the migration against the previous seeds. I.e., by checking out main and starting the Hasura cluster.

I don't see any reason why this migration would fail in staging/prod, but if we wanted to be extra careful i believe we could do:
```
BEGIN;
alter table "public"."moped_proj_personnel" drop column "role_id" cascade;
ROLLBACK;
```

Reply:
I also don't see a reason why this would fail, but it could be a good use case for the using a test instance populated with staging data. But, I can't remember if that is an option at the moment. 

Do you remember @frankhereford? Definitely don't think it is worth extra effort right this moment if not.

Reply:
I'm feeling pretty confident that it will not fail .. I think it will compile and execute, but what I'm wondering if the `cascade` would interact with any foreign key relationships and cause records in related tables to be deleted. To be honest, I really don't know how that would work, and I think Mike's inclination to test this statement with a database populated with production data is good idea. However, I 100% defer to John and his experiences while putting together this PR.

@johnclary I would be super happy to do that test if it'd be helpful -- I can reuse the `reload_db.sh` script from the recent geography work to spin one up.

Reply:
Thanks, y'all. I will run a local "production" test just to be safe. That will definitely be worth the time vs having to fix this after merge.

Wrt to `cascade`, in the case of a dropped column, I don't _think_ it has any relation to the `on update` and `on delete` actions defined in the foreign key constraints, but it will be interesting to test on prod data.

Here's a bit of docs from the [pg tutorial](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-drop-column/):

> If the column that you want to remove is used in other database objects such as [views](https://www.postgresqltutorial.com/postgresql-views/), [triggers](https://www.postgresqltutorial.com/postgresql-triggers/), [stored procedures](https://www.postgresqltutorial.com/postgresql-stored-procedures/), etc., you cannot drop the column because other objects are depending on it. In this case, you need to add the CASCADE option to the DROP COLUMN clause to drop the column and all of its dependent objects.